Examples/Swift/LiveDataExample.swift

```diff
@@ -0,0 +1,49 @@
+import Mapbox
+
+@objc(LiveDataExample_Swift)
+
+class LiveDataExample: UIViewController, MGLMapViewDelegate {
+    
+    var source : MGLShapeSource!
+    var timer = Timer()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds,
+                                 styleURL: MGLStyle.darkStyleURL(withVersion: 9))
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
```

Comment:
Same as in the Objective-C example, we should add a comment above this line that [matches this one](https://github.com/mapbox/ios-sdk-examples/blob/master/Examples/Swift/DrawingACustomMarkerExample.swift#L21).